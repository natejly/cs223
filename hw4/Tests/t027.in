P1,2,3
P2,3,2

P1 P2
