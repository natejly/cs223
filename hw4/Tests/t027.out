P1 2.0 - P2 1.0
